### PR TITLE
Release 1.22.0/0.47.0/0.16.0/0.2.0

### PR DESCRIPTION
**Pull request by MadVikingGod**
### Added

- Add `SDK.Shutdown` method in `"go.opentelemetry.io/contrib/config"`. (#4583)
- `NewSDK` in `go.opentelemetry.io/contrib/config` now returns a configured SDK with a valid `TracerProvider`. (#4741)

### Changed

- The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/github.com/emicklei/go-restful/otelrestful` are upgraded to v1.20.0. (#4320)
- The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin` are upgraded to v1.20.0. (#4320)
- The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux` are upgraded to v1.20.0. (#4320)
- The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/github.com/labstack/echo/otelecho` are upgraded to v1.20.0. (#4320)
- The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/gopkg.in/macaron.v1/otelmacaron` are upgraded to v1.20.0. (#4320)
- The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace` are upgraded to v1.20.0. (#4320)
- The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace/example` are upgraded to v1.20.0. (#4320)
- The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp/example` are upgraded to v1.20.0. (#4320)
- The semantic conventions used by `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp`are upgraded to v1.20.0. (#4320)
- Updated configuration schema to include `schema_url` for resource definition and `without_type_suffix` and `without_units` for the Prometheus exporter. (#4727)
- The semantic conventions used by the `go.opentelemetry.io/contrib/detectors/aws/ecs` resource detector are upgraded to v1.24.0. (#4803)
- The semantic conventions used by the `go.opentelemetry.io/contrib/detectors/aws/lambda` resource detector are upgraded to v1.24.0. (#4803)
- The semantic conventions used by the `go.opentelemetry.io/contrib/detectors/aws/ec2` resource detector are upgraded to v1.24.0. (#4803)
- The semantic conventions used by the `go.opentelemetry.io/contrib/detectors/aws/eks` resource detector are upgraded to v1.24.0. (#4803)
- The semantic conventions used by the `go.opentelemetry.io/contrib/detectors/gcp` resource detector are upgraded to v1.24.0. (#4803)
- The semantic conventions used in `go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-lambda-go/otellambda/test` are upgraded to v1.24.0. (#4803)

### Fixed

- Fix `NewServerHandler` in `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc` to correctly set the span status depending on the gRPC status. (#4587)
- The `stats.Handler` from `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc` now does not crash when receiving an unexpected context. (#4825)
- Update `go.opentelemetry.io/contrib/detectors/aws/ecs` to fix the task ARN when it is not valid. (#3583)
- Do not panic in `go.opentelemetry.io/contrib/detectors/aws/ecs` when the container ARN is not valid. (#3583)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [1.22.0/0.47.0/0.16.0/0.2.0] - 2024-01-18
+
 ### Added
 
 - Add the new `go.opentelemetry.io/contrib/instrgen` package to provide auto-generated source code instrumentation. (#3068, #3108)
```
Comment (pellared) [inline CHANGELOG.md:15]: This line must be moved to unreleased.

Comment (pellared) [reply CHANGELOG.md]: https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4827/commits/1cd1f7075b29a312e214e7f96466ad81cc609415

**Review by pellared — APPROVED** [review]

**Review by dashpole — APPROVED** [review]

**Comment by pellared** [comment]
@MadVikingGod why https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4827/commits/ca5cb71231209bdf080820f75b38128edee42f1b? Instrgen is not released.

**Review by pellared — CHANGES_REQUESTED** [review]
instrgen is not released

**Comment by pellared** [comment]
> > instrgen is not released. It is part of `excluded-modules:`
> 
> Should it be in our changelog at all then? Or are we going to keep floating it to the unreleased every cycle?

As for me you can remove it. If we decide to add it at some point we can use the following query to find related PRs: https://github.com/open-telemetry/opentelemetry-go-contrib/pulls?q=is%3Apr+label%3A%22area%3A+instrgen%22+

**Review by MrAlias — APPROVED** [review]

**Comment by MadVikingGod** [comment]
> We can consider adding #4825. But only if we do it ASAP as I do not want to delay the release.

Agreed, it is now included.

File: CHANGELOG.md (hunk 1 above)
Comment (MadVikingGod) [reply CHANGELOG.md]: I misunderstood this comment, and didn't notice you had pushed the change you intended.  I reverted it.

**Review by pellared — APPROVED** [review]
I updated the PR description.

```
- The `stats.Handler` from `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc` now does not crash when receiving an unexpected context. (#4825)
```

was missing.

**Pull request merged**